src/main/java/bg/libapp/libraryapp/event/UpdateActiveStatusBookAuditEvent.java

```diff
@@ -0,0 +1,48 @@
+package bg.libapp.libraryapp.event;
+
+import bg.libapp.libraryapp.model.entity.Book;
+
+public class UpdateActiveStatusBookAuditEvent extends BaseBookAuditEvent {
+    private String fieldName;
```

Comment:
extract the fields in another base class, that extends BaseBookAuditEvent and make all the statusBookAuditEvent classes to extend the new one